bedrock/base/templates/includes/global-nav.html

```diff
@@ -0,0 +1,279 @@
+<nav role="navigation" class="moz-global-nav" id="moz-global-nav">
+    <div class="nav-horizontal-menu">
+
+      <button type="button" class="nav-button-menu nav-hidden" id="nav-button-menu" aria-controls="moz-global-nav-drawer">
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 25" width="12" height="25">
+          <title>{{_('Menu')}}</title>
+          <g>
+            <path class="p1" fill="#231f20" d="M3 0h4.62v5.55H3zM3"/>
```

Comment:
In Chrome I see these SVG paths are throwing an error `Error: <path> attribute d: Unexpected end of attribute. Expected number, "…0h4.62v5.55H3zM3".` I had to hack this SVG by hand, so I probably screwed up somewhere. I may need to see if I can get a version of the image as three separate paths.